bin/BuildPackages.sh

```diff
@@ -279,6 +381,25 @@ build_one_package() {
   ) || build_fail
 }
 
+
+### The main function to be run, its output is going to be logged.
+build_packages() {
+
+notice "Using GAP location: $GAPDIR"
+echo ""
+
+set_packages
+set_build_flags_for_32_bit
+set_make
```

Comment:
You always needs to invoke these. So why do you put them into functions and only call them here?